If you are not using the Xaw3d widget set, remove this patch

--- src/Velvet/Makefile.orig	Wed Dec 16 21:04:49 1998
+++ src/Velvet/Makefile	Wed Dec 16 21:03:41 1998
@@ -18,7 +18,7 @@
 CFLAGS	= $(CCOPTS) -I$(X11INC) -I$(TOPDIR)/include
 LDFLAGS	= $(LDOPTS) -L$(X11LIB)
 LIBS	= $(LIBWIDGETS) $(LIBGEN) $(LIBGEOMPACK) $(LIBFELT) $(LIBELT) $(LIBMTX)\
-	  -lXaw -lXmu -lXt -lSM -lICE -lXext -lX11 -lm
+	  -lXaw3d -lXmu -lXt -lSM -lICE -lXext -lX11 -lm
 OBJS	= analysis.o animate.o apply.o bivar.o callbacks.o canvas.o\
 	  colors.o constraint.o contour.o draw3d.o drawingshell.o\
 	  element.o elementlist.o elementop.o eps.o error.o field.o file.o\
